doc/source/contribute-baselines.rst

```diff
@@ -0,0 +1,42 @@
+Contribute Baselines
+====================
+
+Do you have a new federated learning paper and want to add a new baseline to Flower? Or do you want to add a new experiment to an already existing baseline paper?
+Great, we really appreciate your contribution.
+
+The goal of creating baselines is to reproduce the experiments in the paper for better comparison to new strategies, datasets, federated pipelines, and so on. 
+
+Before you start to work on a new baseline or experiment please check the `Flower Issues <https://github.com/adap/flower/issues>`_ or `Flower Pull Requests <https://github.com/adap/flower/pulls>`_ if someone else is already working on it. Please open a new issue if you are planning to work on a new baseline or experiment with a short description of the corresponding paper and the experiment you want to contribute.
+
+
+Requirements
+------------
+
+A contribution is really easy. You only have to make sure that your federated learning experiments are running with Flower. As soon as you have created an experiment with Flower included you can contribute it. 
+
+It is recommended to use `Hydra <https://hydra.cc/>`_ to execute the experiment but not necessary. 
+
+Please make sure that you add your baseline or experiment to the corresponding directory as explained in `Execute Baseline <https://flower.dev/docs/execute-baseline.html>`_. Also add a :code:`README.md` with a short explanation of the paper and a detailed description of how to execute the baseline. 
+Please also check if all requirements or library are listed in the :code:`pyproject.toml` or the :code:`requirements.txt` available in the directory `flower/baselines/flwr_baselines <https://github.com/adap/flower/blob/main/baselines>`_. If the requirement or library is not yet listed please add it. If the versions are different please make your that your baseline is running with the already existing version listed in :code:`pyproject.toml` or the :code:`requirements.txt`.
+
+The experiment also needs to contain a file with an automatic downloader of the dataset. This can be included in one of the files or as an extra file.
+
+At the end please also add a plot the results of the experiment that are performed by your code to the experiment directory as well as to the :code:`README.md`. This will help to better understand the experiment and allow the user to figure out any errors.  
```

Comment:
```suggestion
Finally, please also add plots of the results of the experiments that are performed by your code to the experiment directory and include them in :code:`README.md`. This will help others to better understand the experiment and allow them to quickly recognise your contributions.
```